EIPS/eip-draft_multiple_contenthash_records.md

```diff
@@ -0,0 +1,79 @@
+---
+eip: <to be assigned>
+title: Multiple contenthash records for ENS
+author: Filip Štamcar (@filips123)
+discussions-to: https://github.com/ethereum/EIPs/issues/2393
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-18
+requires: 1577
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+This EIP introduces support for multiple `contenthash` records for ENS. It allows hosting a website on multiple distributed systems and accessing it from the same ENS domain. It is an extension of the EIP 1577.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+
+Many applications are resolving ENS names to content hosted on distributed systems. To do this, they use `contenthash` record from ENS domain to know how to resolve names and which distributed system should be used.
+
+However, the domain can store only one `contenthash` record which means that the site owner needs to decide which hosting system to use. Because there are many ENS-compatible hosting systems available (IPFS, Swarm, recently Onion and ZeroNet), and there will probably be even more in the future, lack of support for multiple records could become problematic. Instead, domains should be able to store multiple `contenthash` records to allow applications to resolve to multiple hosting systems.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+A resolver SHOULD store `contenthash` records in `mapping`, containing `protoCode` as key and whole `contenthash` (including `protoCode`) as value. Default `contenthash` SHOULD be stored as empty `bytes` key. Alternatively, it MAY use other storage types, but public interface MUST be the same as described in this EIP.
```

Comment:
Consider using `**SHOULD**` rather than `SHOULD` for consistency across this repository.  Same for all of the other RFC-2119 keywords in the document.